### PR TITLE
Fix | PanelModal

### PR DESCRIPTION
**Pull request by Yopo-Nicolas**

### CODE REVIEW

File: src/Components/PanelModal/PanelModalDrawer/PanelModalDrawer.module.scss [hunk 1]
```diff
@@ -28,12 +30,15 @@
...
-    font-size: 24px;
-    line-height: 28px;
+  .contentTitle {
+    padding: 0 36px;
+    display: flex;
+    flex-direction: column;
+    gap: 6px;
+
+    .subTitle {
+      color: $sf-grey-main-light;
```
Comment (martinweingart) [inline src/Components/PanelModal/PanelModalDrawer/PanelModalDrawer.module.scss:40]: Same as before

File: src/Components/PanelModal/PanelModalDialog/PanelModalDialog.module.scss [hunk 2]
```diff
@@ -33,12 +34,15 @@
...
       padding: 0 36px;
-      font-weight: 500;
-      font-size: 24px;
-      line-height: 28px;
+      display: flex;
+      flex-direction: column;
+      gap: 6px;
+
+      .subTitle {
+        color: $sf-grey-main-light;
```
[the remark this answers is not in the document]
Comment (Yopo-Nicolas) [reply src/Components/PanelModal/PanelModalDialog/PanelModalDialog.module.scss]: Done

File: src/Components/PanelModal/PanelModalDrawer/PanelModalDrawer.module.scss (hunk 1 above)
Comment (Yopo-Nicolas) [reply src/Components/PanelModal/PanelModalDrawer/PanelModalDrawer.module.scss]: Done

**Review by martinweingart — APPROVED** [review]
LGTM

**Pull request merged**